support/export/rmtab.c

```diff
@@ -96,3 +97,99 @@ rmtab_read(void)
 	}
 	return 0;
 }
+
+/*
+ * Check if an rmtab_path is exported given an export_path from etab file.
+ * The path is considered exported if it is an exact match or is part of
+ * a path nested under an export
+ */
+static int
+is_exported(const char* rmtab_path, const char* export_path) {
+	while(*export_path) {
+		if (*export_path++ != *rmtab_path++)
+			return 0;
+	}
```

Comment:
I think this can be done using strstr(rmtab_path, export_path), and then checking if the return value is == rmtab_path?